contributing.qmd

```diff
@@ -2,63 +2,90 @@
-* Landing pages with no code should be use [quarto markdown (`.qmd`)](https://quarto.org/docs/authoring/markdown-basics.html).
-* Pages with executable code should be [iPython Notebooks (`.ipynb`)](https://ipython.org/notebook.html)
+1. This guide intends to be opinionated but acknowledges no one-size-fits-all solution.
+2. This guide should provide the best information and guidance available but acknowledge that experts develop many existing resources. Those resources should be linked as appropriate.
+
+## Additional Criteria
+
+- All examples should use open data. If an example uses data from NASA Earthdata, it must include an example of providing credentials ([Earthdata registration](https://urs.earthdata.nasa.gov/users/new) is available to anyone).
+- Landing pages with no code should use [quarto markdown (`.qmd`)](https://quarto.org/docs/authoring/markdown-basics.html).
+- Pages with executable code should be [Jupyter Notebooks (`.ipynb`)](https://ipython.org/notebook.html).
 
 ## Code of Conduct
 
-* Be inclusive, respectful and understanding of others' backgrounds and contexts.
-* Look for and foster diverse perspectives.
-* If you experience any harmful behavior, please contact aimee@developmentseed.org or alex@developmentseed.org.
+- Be inclusive, respectful, and understanding of others' backgrounds and contexts.
+- Look for and foster diverse perspectives.
+- If you experience any harmful behavior, please get in touch with [Aimee](mailto:aimee@developmentseed.org) or [Alex](mailto:alex@developmentseed.org).
+
+## Bug Reporting & Feature Requests
+
+Before submitting a bug report or a feature request, please start a [GitHub Discussion](https://github.com/developmentseed/cloud-optimized-geospatial-formats-guide/discussions) to see if the issue has already been addressed or if it can be resolved through discussion.
 
-## How to contribute
+### General Steps
 
-### 0. General
+1. Fork the repository.
+2. Clone your fork locally.
+3. Create a new branch for your changes.
+4. Make your changes and use `quarto preview` to make sure they look good.
+5. Open a pull request.
 
-Fork the repository, make changes, use `quarto preview` to make sure the changes look good and open a pull request.
+Once the pull request is opened, and the GitHub `preview.yml` workflow runs ("Deploy PR previews"), you should have a preview available for review at `https://developmentseed.org/cloud-optimized-geospatial-formats-guide/pr-preview/pr-<YOUR-PR-NUMBER-HERE>`. A bot will comment on your PR when the PR preview is ready.
 
-Once the pull request is opened, and the github `preview.yml` workflow runs ("Deploy PR previews"), you should have a preview available for review at https://developmentseed.org/cloud-optimized-geospatial-formats-guide/pr-preview/pr-<YOUR-PR-NUMBER-HERE>.
+### Specific Contributions
 
-### 1. Adding a new format
+#### 1. Adding a New Format
```

Comment:
I've done some googling on how to capitalize subtitles and it seems guidance is inconclusive so I guess I don't care which one we go with as long as we are consistent. This almost makes me want to add a guidelines on formatting page in the future.

Reply:
I agree that either is fine but good to be consistent. I think in my own writing I've used lower case after the first word, so we'd probably need to update those files

Reply:
I updated or verified `overview.qmd`, `index.qmd`, `README.md` and `contributing.qmd`